drivers/motor_driver/motor_driver.c

```diff
@@ -29,155 +29,174 @@
 #define ENABLE_DEBUG 0
 #include <debug.h>
 
-int motor_driver_init(motor_driver_t motor_driver)
-{
-    int err = 0;
+/**
+ * @brief Callback to set direction on two directions pins driver
+ *
+ * @param   motor       motor
+ * @param   direction   direction
+ */
+static void _motor_set_two_dirs(const motor_t *motor, motor_direction_t direction);
+
+/**
+ * @brief Callback to set direction on one direction pin driver
+ *
+ * @param   motor       motor
+ * @param   direction   direction
+ */
 
-    assert(motor_driver < MOTOR_DRIVER_NUMOF);
+static void _motor_set_one_dir(const motor_t *motor, motor_direction_t direction);
 
-    const motor_driver_config_t *motor_driver_conf = \
-        &motor_driver_config[motor_driver];
+static void _motor_brake_two_dirs(const motor_t *motor, bool brake);
+static void _motor_brake_one_dir_brake(const motor_t *motor, bool brake);
 
-    pwm_t pwm_dev = motor_driver_conf->pwm_dev;
-    pwm_mode_t mode = motor_driver_conf->pwm_mode;
-    uint32_t freq = motor_driver_conf->pwm_frequency;
-    uint16_t resol = motor_driver_conf->pwm_resolution;
+int motor_driver_init(motor_driver_t *motor_driver, const motor_driver_params_t *motor_driver_params)
```

Comment:
```suggestion
int motor_driver_init(motor_driver_t *motor_driver,
                      const motor_driver_params_t *motor_driver_params)
```

or may be you can shorten one of the params names

```suggestion
int motor_driver_init(motor_driver_t *motor_driver, const motor_driver_params_t *params)
```

Reply:
done